Source/AudibleUtilities/ApiExtended.cs

```diff
@@ -119,32 +120,33 @@ private async Task<List<Item>> getItemsAsync(LibraryOptions libraryOptions, bool
 		{
 			var items = new List<Item>();
 
-			Serilog.Log.Logger.Debug("Begin library scan");
+			Serilog.Log.Logger.Debug("Beginning library scan.");
 
 			List<Task<List<Item>>> getChildEpisodesTasks = new();
 
-			int count = 0;
+			int count = 0, maxConcurrentEpisodeScans = 10;
```

Comment:
Let's drop this to 5. Ten would *probably* be ok but I want to be conservative. I live in fear of amazon's reputation for IP banning. I have a very long history with site scrapping and they are infamous. Granted, we're piggy backing on audible's mobile api instead of scraping, but I still worry.

In the very first version of what would eventually be Libation, I routed all calls through a "Humanizer" which would actually add a random wait time between each call. That proved to be overkill but it shows my commitment to the idea.

Reply:
Done